File: week5/app/src/main/java/com/senecacollege/week5/MainActivity.java
package com.senecacollege.week5;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    Button increaseButton;
    Button decreaseButton;
    TextView counterTextView;
    int counter;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        increaseButton = findViewById(R.id.increaseButton);
        decreaseButton = findViewById(R.id.decreaseButton);
        counterTextView = findViewById(R.id.counter);

        counter = 0;

        increaseButton.setOnClickListener(view -> {
            counter += 1;
            counterTextView.setText(String.valueOf(counter));
        });

        decreaseButton.setOnClickListener(view -> {
            counter -= 1;
            counterTextView.setText(String.valueOf(counter));
        });
    }
}

File: KotlinCalculator/app/src/main/java/com/senecacollege/kotlincalculator/MainActivity.kt
package com.senecacollege.kotlincalculator

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.Toast


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val addButton = findViewById<Button>(R.id.addButton);
        val subButton = findViewById<Button>(R.id.subButton);


        val editNo1 = findViewById<EditText>(R.id.editNo1);
        val editNo2 = findViewById<EditText>(R.id.editNo2);

        addButton.setOnClickListener {
            if (editNo1.text.toString().isNotEmpty() && editNo2.text.toString().isNotEmpty()) {

                val no1 = editNo1.text.toString().toInt();
                val no2 = editNo2.text.toString().toInt();

                val sum = no1 + no2;

                Toast.makeText(this, "The sum is $sum", Toast.LENGTH_LONG).show();

            } else {
                Toast.makeText(this, "Please fill all fields", Toast.LENGTH_LONG).show();
            }
        }
        subButton.setOnClickListener {
            if (editNo1.text.toString().isNotEmpty() && editNo2.text.toString().isNotEmpty()) {

                val no1 = editNo1.text.toString().toInt();
                val no2 = editNo2.text.toString().toInt();

                val sub = no1 - no2;

                Toast.makeText(this, "The difference is $sub", Toast.LENGTH_LONG).show();

            } else {
                Toast.makeText(this, "Please fill all fields", Toast.LENGTH_LONG).show();
            }
        }
    }
}